pkgs/top-level/python-packages.nix

```diff
@@ -6987,6 +6987,13 @@ in {
     inherit (pkgs) protobuf;
   };
 
+  protobuf4_21 = callPackage ../development/python-modules/protobuf {
+    disabled = isPyPy;
+    # If a protobuf upgrade causes many Python packages to fail, please pin it here to the previous version.
+    doCheck = !isPy3k;
+    protobuf = pkgs.protobuf3_21;
+  };
+
```

Comment:
This is not allowed in pythoNPackages. Please do this locally in the package with python package set override.

Reply:
Done.